Sources/Basics/Observability.swift

```diff
@@ -350,9 +351,10 @@ public struct Diagnostic: Sendable, CustomStringConvertible {
         }
 
         let message: String
-        // FIXME: this brings in the TSC API still
-        // FIXME: string interpolation seems brittle
-        if let diagnosticData = error as? DiagnosticData {
+            // FIXME: this brings in the TSC API still
+            // FIXME: string interpolation seems brittle
+        if let diagnosticData = error as? DiagnosticData
+        {
```

Comment:
Re-formatting here seems unnecessary.

```suggestion
        // FIXME: this brings in the TSC API still
        // FIXME: string interpolation seems brittle
        if let diagnosticData = error as? DiagnosticData {
```